src/backend/d3d12/D3D12Backend.h

```diff
@@ -109,32 +110,42 @@ namespace d3d12 {
 
             ComPtr<ID3D12Device> GetD3D12Device();
             ComPtr<ID3D12CommandQueue> GetCommandQueue();
-            ComPtr<ID3D12CommandAllocator> GetPendingCommandAllocator();
-            ComPtr<ID3D12GraphicsCommandList> GetPendingCommandList();
-            D3D12_CPU_DESCRIPTOR_HANDLE GetCurrentRenderTargetDescriptor();
+            CommandAllocatorManager* GetCommandAllocatorManager();
```

Comment:
Like we discussed offline, it sounds simpler and less error-prone to request a new (or current?) "pending command list" instead of a manipulating command allocators directly. With this GetCommandAllocatorManager doesn't need to be exposed anymore.